backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -137,4 +137,21 @@
         <tagDatabase tag="db_version_2.4"/>
     </changeSet>
 
+    <changeSet id="add-workspace-id-to-project" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="project" columnName="workspace_id"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="project">
+            <column name="workspace_id" type="varchar(255)"/>
```

Comment:
The new owrkspace id is an int, actually

Reply:
Will change

Reply:
We can also rename `workspace` to `workspace_name`